test/Microsoft.Data.Sqlite.Tests/SqliteCommandTest.cs

```diff
@@ -171,6 +171,63 @@ public void Prepare_throws_when_command_text_contains_dependent_commands()
         }
 
         [Fact]
+        public void Multiple_command_executes_works()
+            => Multiple_command_executes_works((SqliteCommand command) =>
+            {
+                command.CommandText = string.Empty;
+                command.CommandText = "INSERT INTO Data (Value) VALUES (@value);";
+            });
+
+        [Fact]
+        public void Multiple_command_executes_works_2()
+            => Multiple_command_executes_works((SqliteCommand command) =>
+            {
+                command.CommandText = "INSERT INTO Data (Value) VALUES (@value);";
+            });
+
+        [Fact]
+        public void Multiple_command_executes_works_3()
+            => Multiple_command_executes_works((SqliteCommand command) => { });
+
+        private void Multiple_command_executes_works(Action<SqliteCommand> action)
+        {
+            const int INSERTS = 3;
+
+            using (var connection = new SqliteConnection("Data Source=:memory:"))
+            {
+                connection.Open();
+
+                using (var command = connection.CreateCommand())
+                {
+                    command.CommandText = "CREATE TABLE Data (ID integer PRIMARY KEY, Value integer);";
+                    command.ExecuteNonQuery();
+
+                    command.CommandText = "INSERT INTO Data (Value) VALUES (@value);";
+                    var valueParam = new SqliteParameter { ParameterName = "@value" };
+                    command.Parameters.Add(valueParam);
+
+                    for (var i = 0; i < INSERTS; i++)
+                    {
+                        action(command);
+
+                        valueParam.Value = i;
```

Comment:
We may also want to add a test that using the same parameter value (i.e. not setting `SqliteParameter.Value`) re-uses the value for each execution.